userspace/sysdig/csysdig.cpp

```diff
@@ -700,17 +700,6 @@ sysdig_init_res csysdig_init(int argc, char **argv)
 				bpf = true;
```

Comment:
I think this condition should become `if(probe)`, because we now want to set `bpf=true` even when `SYSDIG_BPF_PROBE` is an empty string, right? I see you did that on sysdig.cpp.

Reply:
Yep, exactly! Forgot about that :/ Thanks!